s32/drivers/s32k3/Icu/CMakeLists.txt

```diff
@@ -12,3 +12,8 @@ zephyr_library_sources_ifdef(CONFIG_NXP_S32_WKPU
     src/Wkpu_Ip.c
     src/Wkpu_Ip_Irq.c
 )
+
+zephyr_library_sources_ifdef(CONFIG_PWM_NXP_S32_EMIOS_PULSE_CAPTURE
```

Comment:
cannot we have a single Kconfig option for this PWM driver and use the `CONFIG_PWM_CAPTURE` to conditionally build capture support?

Reply:
I've updated